static/docs/get-started/agenda.md

```diff
@@ -1,36 +1,35 @@
 # Agenda
 
-In the next few sections we will build a simple natural language processing
-(NLP) project from scratch. If you'd like to get the final result or have some
-issues along the way, you can download the fully reproducible
-[GitHub project](https://github.com/iterative/example-get-started) with:
+In the next few sections we'll build a simple natural language processing (NLP)
+project from scratch. If you'd like to get the final result or have any issues
+along the way, you can download the fully reproducible
+[GitHub project](https://github.com/iterative/example-get-started) by running:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 ```
 
-Otherwise, bear with us and we will introduce the basic DVC concepts and get to
-the same result together!
+Otherwise, bear with us and we'll introduce some basic DVC concepts to get the
+same results together!
```

Comment:
Not crazy about the "together" part now that I notice it. What do you think?

Reply:
I think it's good! Makes it sound collaborative